### ISSUE TITLE
关于 LabelSmoothing 的一些问题

### ISSUE THREAD

**Issue by chenjunyi1999** [taoztw/Transformer#2]
你好，很感谢你的代码， 在里面学到了很多，但是关于LabelSmoothing我感觉有一点问题。
` return self.criterion(x, Variable(true_dist, requires_grad=False))` 中， 我感觉因为使用`nn.KLDivLoss(reduction='sum')`所以理论上需要对x取log。但是我尝试修改之后发现loss会变成nan。请问这个需要怎么解决。能否指点一下。

**Comment by taoztw** [comment]
您好，`model.py`文件中class `Generator`实现了decoder输出到词表的映射。`F.log_softmax(self.proj(x), dim=-1)`所以计算损失处模型输出x  已经进行log运算了。

**Comment by chenjunyi1999** [comment]
感谢。之前看的不仔细，不好意思哈。
